README.md

```diff
@@ -1,11 +1,24 @@
-#Vote 2016
+# Vote 2016
 
-To get the SASS working, run `sass --watch assets/stylesheets/sass:assets/stylesheets/output` from the project directory.
+**This repository has been deprecated. Please use the [`18F/vote-2016`] [18f-fork]
+fork to follow development for this project or to file any issues. You must open
+all pull requests against the `master` branch on the `18F/vote-2016` repository.**
+
+[18f-fork]: https://github.com/18F/vote-2016 "GitHub 18F Vote 2016 Fork"
+
+To get the SASS working, run the following command from the project directory:
+
+```sh
+sass --watch assets/stylesheets/sass:assets/stylesheets/output
+```
 
 ### Public domain
 
 This project is in the worldwide [public domain](LICENSE.md). As stated in [CONTRIBUTING](CONTRIBUTING.md):
 
-> This project is in the public domain within the United States, and copyright and related rights in the work worldwide are waived through the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
->
-> All contributions to this project will be released under the CC0 dedication. By submitting a pull request, you are agreeing to comply with this waiver of copyright interest.
+> This project is in the public domain within the United States, and copyright
```

Comment:
Hey! I might consider breaking this into two sentences:

"This project is in the public domain within the United States. Worldwide, copyright and related rights are waived through the CC0 1.0 ... dedication."

(The current placement of *worldwide* is throwing me a bit.)